src/components/Main/TvShowPoster.js

```diff
@@ -1,14 +1,32 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import { Link } from 'react-router-dom';
+import { Skeleton } from '../../constants/SkeletonLoading';
 
 export const TVShowPoster = ({ data }) => {
+  const [loading, setLoading] = useState(false);
```

Comment:
I don't think this is the effect we want. This just plays an animation after data has already been loaded.

We want the skeleton to appear before data is fetch and the animation will persist until data is populated